libmscore/system.cpp

```diff
@@ -361,6 +361,106 @@ void System::layoutSystem(qreal xo1)
             }
       }
 
+//---------------------------------------------------------
+//   addBrackets
+//   Add brackets in front of this measure, typically behind a HBox
+//---------------------------------------------------------
+
+void System::addBrackets(Measure* measure)
+{
+      if (_staves.empty())                 // ignore vbox
+            return;
+
+      int nstaves = _staves.size();
+
+      //---------------------------------------------------
+      //  find x position of staves
+      //    create brackets
+      //---------------------------------------------------
+
+      int columns = 0;
+      for (int idx = 0; idx < nstaves; ++idx) {
+            int c = 0;
```

Comment:
what does "c" stands for?

Reply:
See my comment to your previous message. 
Variable names in MuseScore are also a mystery to me